shakedown/dcos/package.py

```diff
@@ -123,7 +121,7 @@ def install_package(
     if wait_for_completion:
         print("\n{}waiting for {} deployment to complete...\n".format(
             shakedown.cli.helpers.fchr('>>'), service_name))
-        if expected_running_tasks > 0:
+        if expected_running_tasks > 0 and service_name is not None:
```

Comment:
So if there is no `DCOS_SERVICE_NAME` we'll not wait for tasks to start? Because there is no endpoint? Is that the idea?